src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -172,10 +172,10 @@ xorGate i1 i2 o = [rise i1, rise i2] ~|~> rise o <> [fall i1, fall i2] ~|~> rise
                <> [rise i1, fall i2] ~|~> fall o <> [fall i1, rise i2] ~|~> fall o
 
 srLatch :: Eq a => a -> a -> a -> CircuitConcept a
-srLatch s r q = complexGate (Var s) (Var r) q
+srLatch s r q = complexGate (Var s) (Var r) q <> mutex s r
```

Comment:
Ahh, right. As I feared. On import to Workcraft of just an `srLatch` or `srLatch2`, this will state that the `never` state can be reached, but will not stop the import.